### DISCUSSION TITLE
Displaying a large number of tracks as an overlay (source data is in GeoJSON format)

### DISCUSSION THREAD

**Discussion by rachellawson** [osmandapp/OsmAnd#18441] [Q&A]
I have a large number of tracks (about 10k ways) in GeoJSON format that I'd LOVE to be able to display as an overlay on the map as I ride.

What steps do I need to follow to convert this data into a suitable format and add to Osmandapp? I feel I'm sooooo close but not quite cracked it yet...

**Comment by mariush444** [comment]
Currentky, there is only one somution -> conversion.
Use ourxmap - import as a track then save as gpx or convert online https://www.gpsvisualizer.com/gpsbabel/
but number of your track is a problem in any case.

**Comment by vshcherb** [comment]
You can convert them on the website into OBF file https://osmand.net/map/ - OBF file would support > 1M points and it's designed to do that.

**Comment by rachellawson** [comment]
For the record, I wrote up how I finally achieved this at https://rachelnorfolk.me/2024/oooh-i-can-ride-down-track if it is of interest to any watchers. Thanks all!
